src/Pages/3DPrintingConsole/3DConsole.js

```diff
@@ -89,6 +89,47 @@ export default class PrintConsole3D extends React.Component {
   }
 
   requestForm (jsonObject, key) {
```

Comment:
can you make requestForm it's own react component? you can send the `jsonObject` as a prop.